installer.sh

```diff
@@ -171,6 +221,14 @@ $ <command> <-h --help> # Prints help for the command
 
 Advanced usage:
 ------------------------------------------------------------------------------------
+## select_browser: ################################################################
+$SELECT_BROWSER_HELP
+####################################################################################
+
```

Comment:
"set_target" is better placed below "packages_install" in the intuative order of things